tutorial.md

```diff
@@ -271,8 +277,26 @@ sudo sh -c 'cat >/etc/cni/net.d/99-loopback.conf <<-EOF
 EOF'
 ```
 
+Copy policy.json
```

Comment:
Remove this bit on the policy.json.  It will be provide via skopeo-containers from above.